src/game/boe.main.cpp

```diff
@@ -264,6 +318,7 @@ void handle_one_event(const sf::Event& event) {
 			change_cursor({event.mouseMove.x, event.mouseMove.y});
 			return;
 
+		// XXX EVENT TYPE DEPRECATED IN SFML 2.5.1
```

Comment:
Another improperly-formatted TODO note.

I'm probably building with a rather older SFML, too, so this is far from a priority.

Reply:
done